convert2rhel/utils.py

```diff
@@ -473,8 +473,16 @@ def download_pkg(
     if ret_code != 0:
         loggerinst.warning(
             "Couldn't download the %s package using yumdownloader.\n"
-            "Output from the yumdownloader call:\n%s" % (pkg, output)
+            " \nOutput from the yum downloader call:\n%s %s" % (pkg, output)
```

Comment:
```suggestion
            " Output from the yumdownloader call:\n%s" % (pkg, output)
```

Reply:
I have went back and fixed the typos in this comment